models.py

```diff
@@ -1401,3 +1403,36 @@ def __init__(self, *args, **kwargs):
 			nonlinearity = ('leaky_relu', 0.01),
 			**kwargs
 		)
+
+
+class OnnxModelWraper(nn.Module):
+	def __init__(self, onnx_model_path, provider=None):
```

Comment:
Please put in `['CUDAExecutionProvider', 'CPUExecutionProvider']` as the default value

Reply:
done